### ISSUE TITLE
Add a splash screen for the Godot Engine.

### ISSUE THREAD

**Issue by drwhut** [drwhut/tabletop-club#67]
**Is your feature request related to a problem? Please describe.**
Right now, the game uses the default splash screen used by the Godot Engine. It would be nice to have our own custom version!

**Describe the solution you'd like**
Design and create a new splash screen representing the Godot Engine, in the style of the game. It should also say something like "Made with the Godot Engine", or something along those lines.

**Describe alternatives you've considered**
None.

**Additional context**
This thread is kind of a continuation of #38, as @leogott posted a proposal for the splash screen in that issue.

**Comment by Herobrine62** [comment]
What if yo create a scene and then put it to the start and make a script to load the game after few seconds

**Comment by drwhut** [comment]
Yeah, that's most likely how I will implement it - the initial boot screen just goes away too quickly, especially so on SSDs.

**Comment by drwhut** [comment]
I've put in a simple splash screen for now, which just fades in and out the normal Godot logo - but I'll keep this issue open just in case anyone wants to have a go at making a customized version of the logo specifically for Tabletop Club!
